### PR TITLE
Spike out a random theme page

### PR DESCRIPTION
**Pull request by johno**

### CODE REVIEW

File: packages/style-guide/src/ColorPalette.js [hunk 1]
```diff
@@ -15,33 +15,33 @@ export const ColorRow = ({ colors, name, omit = ['modes'], ...props }) => {
           flexWrap: 'wrap',
         }}
       >
-        {Object.keys(colors).map(key => {
-          const color = colors[key]
-          if (!color || omit.includes(key)) return false
-          const id = join(name, key)
-          if (typeof color === 'object') {
+        {Object.entries(colors)
+          .sort(([_, colorA]) => (typeof colorA === 'string' ? -1 : 1))
```
Comment (jxnblk) [inline packages/style-guide/src/ColorPalette.js]: Is this alphabetic sort? I sort of liked using `Object.keys` here

Comment (jxnblk) [reply packages/style-guide/src/ColorPalette.js]: Nevermind, this is just moving non-objects first, right?

**Comment by jxnblk** [comment]
:shipit:

File: packages/style-guide/src/ColorPalette.js (hunk 1 above)
Comment (johno) [reply packages/style-guide/src/ColorPalette.js]: Yeah, bumping up the direct values from scales/objects.

**Pull request merged**